packages/hardhat/hardhat.config.ts

```diff
@@ -76,6 +78,18 @@ const config: HardhatUserConfig = {
       url: `https://polygon-mumbai.g.alchemy.com/v2/${providerApiKey}`,
       accounts: [deployerPrivateKey],
     },
+    zkSyncTestnet: {
```

Comment:
> There is another issue. If you want to replicate all the zkSync features (like a Paymaster) locally, you have to run a local zkSync node, which is not the same that running yarn chain. Maybe it's something we should add a note somewhere at least... https://era.zksync.io/docs/tools/hardhat/testing.html

Maybe something like?

```suggestion
    // If you want to run a local node with all zkSyncEra functionalities, check:
    // https://era.zksync.io/docs/tools/hardhat/testing.html
    zkSyncTestnet: {
```